server_admin/topics/events/login.adoc

```diff
@@ -79,6 +79,41 @@ This logging is very useful if you want to use a tool like Fail2Ban to detect if
 is trying to guess user passwords.  You can parse the log file for `LOGIN_ERROR` and pull out the IP Address. Then feed this information
 into Fail2Ban so that it can help prevent attacks.
 
+The Logging Event Listener logs events to the `org.keycloak.events` logger category. By default debug log events are not
+included in server logs. To enable these you can either configure the log level for the `org.keycloak.events` or change
+the log level used by the Logging Event Listener. This is done by editing the `standalone.xml`, `standalone-ha.xml`, or `domain.xml` files.
+
+For example to change the log level add the following:
+
+[source,xml]
+----
+<subsystem xmlns="urn:jboss:domain:logging:...">
+    ...
+    <logger category="org.keycloak.events">
+        <level name="DEBUG"/>
+    </logger>
+</subsystem>
+----
+
+Or to change the log level used by the Logging Event listener add the following:
+
+[source,xml]
+----
+<subsystem xmlns="urn:jboss:domain:keycloak-server:...">
+    ...
+    <spi name="eventsListener">
+      <provider name="jboss-logging" enabled="true">
+        <properties>
+          <property name="success-level" value="info"/>
+          <property name="error-level" value="error"/>
+        </properties>
+      </provider>
+    </spi>
+</subsystem>
+----
+
+Valid values for the log levels are `debug`, `info`, `warn`, `error` and `fatal`.
+
 The Email Event Listener sends an email to the user's account when an event occurs.
```

Comment:
listener; lower case l.

Reply:
Email Event Listener is a name, so should be upper-case